src/Microsoft.TestPlatform.Common/ExtensionDecorators/ExtensionDecoratorFactory.cs

```diff
@@ -16,7 +16,7 @@ public ExtensionDecoratorFactory(IFeatureFlag featureFlag)
 
     public ITestExecutor Decorate(ITestExecutor originalTestExecutor)
     {
-        return _featureFlag.IsSet(FeatureFlag.DISABLE_SERIALTESTRUN_DECORATOR)
+        return _featureFlag.IsSet(FeatureFlag.VSTEST_DISABLE_SERIALTESTRUN_DECORATOR)
```

Comment:
Sorry for mixing changes, but having the names defined as prefix `VSTEST_` and and name `DISABLE_SERIALTESTRUN_DECORATOR` that you actually see here in the code was confusing me on multiple occasions where I forgot to put the `vstest_` before the flag. 

This is not a breaking change, because the flags are string constants so they are emitted directly into the consuming assembly, and the content of this const did not change.